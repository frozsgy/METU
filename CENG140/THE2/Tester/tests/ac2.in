ADD a
ADD b
ADD c
ADD d
ADD e
ADD f
ADD g
ADD h
ADD i
ADD j
ADD k
ADD l
ADD m
ADD n
ADD o
ADD p
ADD r
SUBSCRIBE b 11
SUBSCRIBE b 22
SUBSCRIBE c 33
SUBSCRIBE d 33
SUBSCRIBE e 44
SUBSCRIBE f 44
SUBSCRIBE g 44
SUBSCRIBE h 55
SUBSCRIBE j 11
SUBSCRIBE b 33
SUBSCRIBE b 44
SUBSCRIBE b 55
SEND m 11
SEND p 22
SEND r 33
SEND e 44
SEND h 11
SEND b 11
SEND b 22
SEND b 33
SEND b 44
SEND b 55
UNSUBSCRIBE b 11
UNSUBSCRIBE b 22
UNSUBSCRIBE b 33
UNSUBSCRIBE b 44
UNSUBSCRIBE b 55
SEND b 11
SEND b 22
SEND b 33
SEND b 44
SEND b 55
SUBSCRIBE b 55
SEND a 55
SEND g 55
SEND a 55
DELETEUSER a
DELETEUSER c
