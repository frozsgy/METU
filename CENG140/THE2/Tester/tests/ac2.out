2: 11 12 13 14 15 17
4: 3 8 13
5: 4 9 14
6: 4 9 14
7: 4 9 14 17
8: 5 10 15 17
9:
10: 1 5 6 11
11:
12:
13: 1
14:
15:
16: 2
17: 3
